app/templates/components/email-unsubscribe.hbs

```diff
@@ -0,0 +1,95 @@
+{{#if fetchRepo.isRunning}}
+
+  {{loading-screen center=true margin=true}}
+
+{{else}}
+  <div class="container">
+
+    {{svg-jar "sadmail" class="picture" data-test-email-unsubscribe-sadmail=true}}
+
+    {{#if isError}}
+
+      <h1 class="title" data-test-email-unsubscribe-title>
+        We didn't find the repository <br /> that you wanted to unsubscribe from
+      </h1>
+
+      <p class="description" data-test-email-unsubscribe-description>
+        Please make sure you have access to the repository and try again.
+      </p>
+
+      <div class="controls">
+        <button
+          class="button--white button--hover"
+          onclick={{perform fetchRepo}}
+          data-test-email-unsubscribe-primary-button
+        >
+          Try again
+        </button>
+      </div>
+
+    {{else}}
+
+      <h1 class="title" data-test-email-unsubscribe-title>
+        Unsubscribe from {{#link-to 'repo' repo target="_blank" class="repo-link repo-slug"}}
+          {{repo.formattedSlug}}
+        {{/link-to}}
+      </h1>
+
+      <p class="description" data-test-email-unsubscribe-description>
+        {{#if isSubscribed}}
+          Please confirm that you would like to unsubscribe from build emails for the
+        {{else}}
+          You are unsubscribed from build emails for the
+        {{/if}}
+        <span class="repo-slug repo-slug--bold">{{repo.formattedSlug}}</span>
+        repository.
+
+        {{#if showConfigNote}}
+          If you’re still receiving emails after unsubscribing, please check your
+          <a target="_blank" href="https://github.com/{{repo.slug}}/blob/master/.travis.yml">.travis.yml</a>
```

Comment:
it’s maybe not a big deal but when I tried this just now on a test repository I got a 404 for [this link](https://github.com/backspace/travixperiments-redux/blob/master/.travis.yml) because `primary` is my default branch and `master` doesn’t exist.

Reply:
We had a discussion about this in [#project-unsubscribe](https://travisci.slack.com/archives/CC97UDR8R/p1536230141000100). In short, this page is not aware of a branch name or commit. If we want to make it more flexible, we have to provide a branch name in query parameter. But it may be not enough, because config may be different on a commit level. So pointing to default branch is not a rock-solid solution, but it would work in most cases.

Although, putting the `repo.defaultBranch` instead of `master` may be a good idea 👍 

What do you think?

Reply:
yeah, I think even with that limitation that using the default branch is a better idea, thank you 😀